### ISSUE TITLE
Creating a project within an existing repository causes a fatal git error

### ISSUE THREAD

**Issue by LloydTao** [nuxt/create-nuxt-app#587]
Creating a project within an existing repository renders git unable to track the project in its entirety.

**git add**
```
$ git add .
error: 'project/' does not have a commit checked out
fatal: adding files failed
```

**git status**
```
$ git status
On branch master

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        project/

nothing added to commit but untracked files present (use "git add" to track)
```

An option needs to be added for the creation of the `.git/` directory.

This is likely related  #366, which has been ignored since October 2019:

> _1. Not everyone uses git (shocking, I know)_
> _2. Sometimes you might be creating the project in an existing repository._

**Comment by azrikahar** [comment]
This is resolved by #596 right?

**Comment by LloydTao** [comment]
@azrikahar yeah! i was meant to write `Closes #587` 😁 

![image](https://user-images.githubusercontent.com/41923457/91993938-e5076600-ed2d-11ea-87f6-96a389e46038.png)

closing now.
